presto-main/src/test/java/io/prestosql/metadata/TestInformationSchemaConnector.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.metadata;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.prestosql.Session;
+import io.prestosql.connector.MockConnectorFactory;
+import io.prestosql.plugin.tpch.TpchConnectorFactory;
+import io.prestosql.spi.connector.SchemaTableName;
+import io.prestosql.testing.LocalQueryRunner;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static io.prestosql.testing.TestingSession.testSessionBuilder;
+import static org.testng.Assert.assertEquals;
+
+@Test(singleThreaded = true)
+public class TestInformationSchemaConnector
+{
+    private LocalQueryRunner queryRunner;
+
+    @BeforeTest
```

Comment:
Don't use TestNG's `@BeforeTest`, it's run too early.
Use `@BeforeClass` instead.

(Or, better yet, extend `AbstractTestQueryFramework`)